classes/observer.php

```diff
@@ -61,7 +61,11 @@ public static function activity_completed(\core\event\course_module_completion_u
         $userid = $event->relateduserid;
         $courseid = $event->courseid;
         
-
+        $authogeneratecertificate = $DB->get_record('simplecertificate', ['course' => $courseid], 'autogeneratecertificate');
+        if(!$authogeneratecertificate || $authogeneratecertificate->autogeneratecertificate === 0)
```

Comment:
@sudhakar-cisinlabs Typo in naming: Please use `$autoGenerateCertificate` for the name of the variable. Thanks

Reply:
I have made the changes as per your request.